### ISSUE TITLE
Add issue with blue snow

### ISSUE THREAD

**Issue by tannisroot** [lah7/sims-2-wine-patches#7]
When you have Seasons expansion installed and some snow falls, the snowy ground outside the lot will look blue instead of white:
![Screenshot from 2019-04-06 04-33-58](https://user-images.githubusercontent.com/10602045/55664195-2fcc3800-5833-11e9-840d-551b01ad3239.png)

**Comment by lah7** [comment]
Interesting - I haven't seen this before.

![Screenshot at 2019-04-06 13:11:57](https://user-images.githubusercontent.com/13032135/55669709-873dc880-5872-11e9-837e-e35e7968b8e4.png)

![Screenshot at 2019-04-06 13:41:11](https://user-images.githubusercontent.com/13032135/55669708-8311ab00-5872-11e9-854e-74f54e37ead1.png)

![Screenshot at 2019-04-06 13:42:37](https://user-images.githubusercontent.com/13032135/55669716-9cb2f280-5872-11e9-9106-68c8a1eabf7a.png)

* Wine 4.0 (virtual desktop)
* Sims 2 - Non-origin, retail discs.
  * Settings set to **High** or **Extra Large**.
* Graphics Card: NVIDIA GTX 960
* Driver: 415.27
* Ubuntu 16.04.6

---

What's your environment for comparison?

**Comment by tannisroot** [comment]
* Wine 3.21 (but same with 4.5)
* Sims 2 Ultimate collection
    * I've never seen "Extra Large" option but I have everything maxed out
* Graphics Card: NVIDIA GTX 1070
* Arch Linux
But I know another person who same the exact same issue on Ubuntu.

**Comment by tannisroot** [comment]
Do you mind sending your `-config-log.txt` log?

**Comment by tannisroot** [comment]
Also, do you have any mods installed? If you do, do you mind sending them all in an archive?

**Comment by lah7** [comment]
> I've never seen "Extra Large" option but I have everything maxed out

Sorry, I meant under "Game Settings", like View Distance and Fade Distance:

![image](https://user-images.githubusercontent.com/13032135/55683004-971ce180-5932-11e9-870e-6f49c397f4f5.png)

---

I tested again (since I updated my graphics driver to 418.56 yesterday). I used a fresh "My Documents/EA Games" folder and only have the 2 mod "workarounds" from yourself, and this `userStartup.cheat`:

```
boolprop controlpets true
boolprop force12HrTime true
boolprop useShaders true
boolProp skipTangentsInVertexData true
aging off
```

Tried the **Traveler** family (from the family bin) in **Riverblossem Hills**, same result, no blue snow. Also no problems in another neighbourhood - the **Pleasant** family from **Pleasantview**:

![image](https://user-images.githubusercontent.com/13032135/55683338-6b9bf600-5936-11e9-9197-b2e2e4f84246.png)

As requested, my [config-log.txt](https://github.com/lah7/sims-2-wine-patches/files/3051514/LukeUbuntuDeskt-config-log.txt). I'd be interested which version The Sims 2 reports for the Ultimate Collection (origin).

---
As a test, `boolprop useShaders false` smoothens the snow between the lot's boundaries, but then it's back to red T-poses:

![image](https://user-images.githubusercontent.com/13032135/55683362-c33a6180-5936-11e9-8654-bc590c1a016b.png)
